src/test/compile-fail/impl-duplicate-methods.rs

```diff
@@ -11,7 +11,7 @@
 struct Foo;
 impl Foo {
     fn orange(&self){}
-    fn orange(&self){}   //~ ERROR error: duplicate method
+    fn orange(&self){}   //~ ERROR duplicate associated function
```

Comment:
Ahh, so maybe what this message should really do is say "duplicate method" when we're dealing with methods, and "duplicate associated function" otherwise. I will see if I can make that work.